### ISSUE TITLE
need to investigate dual verbs in verbpdgm26 table

### ISSUE THREAD

**Issue by monzug** [alpheios-project/inflection-tables#332]
Maybe I am applying the math logic to the Greek tables that do not have any logic. anyhow, I do not understand why the subjective dual is also indicative but the contrary is not true.

e.g. Present System Active of Contract Verbs in -όω, verbpdgm26 table:
δηλοῦτον    is indicative 2nd pers sing dual  and others...
δηλῶτον    is indicative and subjective 2nd pers dual and others
![Screen Shot 2020-01-15 at 3 26 00 PM](https://user-images.githubusercontent.com/41396793/72441782-9069a780-37ab-11ea-9575-364a55676c34.png)
![Screen Shot 2020-01-15 at 3 18 36 PM](https://user-images.githubusercontent.com/41396793/72441784-91023e00-37ab-11ea-960f-a170514cb781.png)

**Comment by balmas** [comment]
>Maybe I am applying the math logic to the Greek tables that do not have any logic. 

Hah! I am quite sure that there is no logic in Greek :-P

**Comment by vgorman1** [comment]
No, it definitely has a logic to it.  It is just that the rules are carefully hidden in the trust of a handful of historical linguists.

Vanessa B. Gorman
Professor of Ancient History
Department of History
619 Oldfather Hall
University of Nebraska-Lincoln
Lincoln, NE 68588-0327
https://vgorman1.github.io/

**Comment by monzug** [comment]
same for first person plural subjective:  δηλῶμεν
2nd person plural subjective:  δηλῶτε
